src/modules/message-parsers.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Incoming message parser - parse message based on headers
+ * @param  {object} _msg An amqp.node incoming message
+ * @return {any}      a string, object, number to send. Something stringifiable
+ */
+module.exports.in = (_msg) => {
+  //if sender put a json header, we parse it to avoid the pain for the consumer
+  if (_msg.properties.contentType === 'application/json') {
+    try {
+      return JSON.parse(_msg.content.toString());
+    } catch(e) {}
```

Comment:
I have an issue with this. It can't happen when inside BunnyMQ but it can happen when plugged with another tool that will fill the header and not provide a correct json.

So I'm unable to test it :/

What do you think of removing the try/catch and letting the error bubble-ing up? 

@jkernech @mrister